### DISCUSSION TITLE
fix the mech possesion on v50

### DISCUSSION THREAD

**Discussion by arandomuser0** [winstxnhdw/lc-hax#495] [Ideas]
the ability is not working, after the possesion it stops moving

**Comment by winstxnhdw** [comment]
Is v50 even out yet?

**Reply by joep26020** [reply]
public beta in steam betas

**Reply by winstxnhdw** [reply]
If it’s not yet released, I am not going to do anything about it.

**Reply by arandomuser0** [reply]
zeekers said that its the final version (bug test) until beta, please do it

**Reply by winstxnhdw** [reply]
No. Updating for v50 would break the game for those still in v49. Be patient and wait for its release.

**Reply by arandomuser0** [reply]
but only for old bird it would change nothing for v49

**Reply by winstxnhdw** [reply]
That’s not how it works..

**Reply by winstxnhdw** [reply]
Just realised the beta is released on April Fools. Even more reason not to push an update lmao.

**Reply by joep26020** [reply]
haha true lol i would just wait until april 2nd and see what happens

**Reply by arandomuser0** [reply]
@winstxnhdw so are you able to do it?

**Reply by winstxnhdw** [reply]
wait for v50 to actually be released. are u the only one in the world using lc-hax?

**Reply by arandomuser0** [reply]
ok then

**Comment by arandomuser0** [comment]
take a look on v50

**Reply by iamdogshi** [reply]
the v50 wasnt an april fools thing which is the funny part

**Comment by winstxnhdw** [comment]
Looks like v50 is out. I’ll update tonight.

**Comment by xAstroBoy** [comment]
i fixed it in my fork.

**Reply by blueman5** [reply]
no you didn't, you cant use the radmech ability

**Reply by blueman5** [reply]
same thing as his lc hax

**Reply by xAstroBoy** [reply]
rad mech shooting is host only sadly, you can walk it around but not much unless host.
And yes i did, you didn't check the repo?

**Comment by xAstroBoy** [comment]
Rad mech might be not doable at all, once possessed and unpossessed, it will break unless you are host.

**Reply by winstxnhdw** [reply]
I see a trend where all new mobs are becoming host-only :/ Things aren’t looking good.

**Reply by xAstroBoy** [reply]
Butler is still controllable, just having troubles to do his actions in the game as non-host, but is still possible to manipulate it without it breaking
